.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,7 +12,7 @@ For all pull requests:
 - [ ] The changes are not breaking the build in downstream projects using SNAPSHOT dependencies, GeoWebCache and GeoServer.
 
 The following are required only for core and extension modules (they are welcomed, but not required, for unsupported modules):
-- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
+- [ ] There is a ticket in Jira describing the issue/improvement/feature. (a notable exemptions is, changes not visible to end users). The ticket is for the GeoTools project, if the issue was found elsewhere it's a good practice to link to the origin ticket/issue.
```

Comment:
origin should be original. 

Also I think its called issue in Jira, so perhaps:
```
[ ] There is a issue in Jira for the GeoTools project (i.e. a `GEOT-` ticket) describing the issue/improvement/feature. (a notable exemptions is, changes not visible to end users). If the issue was found elsewhere it's a good practice to link to the original ticket/issue from the PR as well as the GeoTools issue.
```

Reply:
Okay we can make the language consistent:

- "issue"
- "issues" describe the "bug/task/new feature".